hackathon-topics.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "title": "Example Topic",
+        "description": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+        "host": "Some Guy",
+        "homepage": "https://github.com/cko/nixcon2017",
+        "skillLevel": 1,
+        "skillLevelTo": 3
```

Comment:
As this hackathon might cover topics from Nix over containers to language specific builds I think it would make sense to list required previous knowledge instead of a vague skill level

Reply:
After thniking about it: I would say that the specific requirements should be clear from the description, the `skillLevel` is thought as guideline for beginners and newbies and thus has another goal.